.editorconfig

```diff
@@ -1,2 +1,3 @@
 [*.{kt,kts}]
 disabled_rules=import-ordering
+max_line_length = 100
```

Comment:
Why so short? 120 columns is OK to me, can you raise it?

Reply:
I added this config because of this failure: https://github.com/DanySK/upgradle/runs/704011064 (before this file was updated!)
It seems striker CI had a different opinion about line-length than ktlint - so I thought you specified striker and forgot to set it in this file. 
After setting this here, I needed to fix all the other places you saw, probably undetected issues because striker just runs on the diff of files?

I'm more than happy to raise to 120 and revert the linebreaks (as far as possible)

Reply:
120 is fine. I won't go over 130.

Reply:
However that's a strange failure on stickler... Most probably a failure of mine at configuring

Reply:
Cool. I'll set it to 120 and revert the linebreaks. (Tonight I hope)